wikimedia/wikipedia-ios
# Add time warning to image recs

**Phabricator:** https://phabricator.wikimedia.org/T364049

### Notes
* This PR adds a timed warning to the "Yes" button

### Test Steps
1. On an image recommendation, tap the yes button quickly as the image appears
2. Verify the warning is shown
3. Try again. The warning should not appear as probably 5 seconds have already passed
4. Verify the event is being logged correctly
